showcases/shared/tag.json

```
@@ -118,70 +113,70 @@
 				"props": {
 					"component": "button"
 				},
-				"code": {
-					"angular": "<db-tag><db-button>Interactive Button</db-button></db-tag>",
-					"html": "<div class=\"db-tag\"><button class=\"db-button\">Interactive Button</button></div>",
-					"react": "<DBTag><DBButton>Interactive Button</DBButton></DBTag>",
-					"vue": "<DBTag><DBButton>Interactive Button</DBButton></DBTag>"
-				}
+				"children": [
+					{
+						"name": "button"
+					}
+				]
 			},
 			{
 				"name": "Interactive Link",
 				"props": {
 					"component": "link"
 				},
-				"code": {
-					"angular": "<db-tag><db-link>Interactive Link</db-link></db-tag>",
-					"html": "<div class=\"db-tag\"><a class=\"db-link\">Interactive Link</a></div>",
-					"react": "<DBTag><DBLink>Interactive Link</DBLink></DBTag>",
-					"vue": "<DBTag><DBLink>Interactive Link</DBLink></DBTag>"
-				}
+				"children": [
+					{
+						"name": "link"
+					}
+				]
 			},
 			{
 				"name": "Interactive Checkbox",
 				"props": {
 					"component": "checkbox"
 				},
-				"code": {
-					"angular": "<db-tag><db-checkbox>Interactive Checkbox</db-checkbox></db-tag>",
-					"html": "<div class=\"db-tag\"><input type=\"checkbox\" class=\"db-checkbox\" id=\"checkbox-element\" /><label for=\"checkbox-element\">Interactive Checkbox</label></div>",
-					"react": "<DBTag><DBCheckbox>Interactive Checkbox</DBCheckbox></DBTag>",
-					"vue": "<DBTag><DBCheckbox>Interactive Checkbox</DBCheckbox></DBTag>"
-				}
+				"children": [
+					{
+						"name": "checkbox"
+					}
+				]
 			},
 			{
 				"name": "Interactive Radio 1",
 				"props": {
 					"component": "radio",
 					"identifier": "radio01"
 				},
-				"code": {
-					"angular": "<db-tag><db-radio name=\"radio01\">Interactive Radio 1</db-radio></db-tag>",
-					"html": "<div class=\"db-tag\"><input type=\"radio\" class=\"db-checkbox\" id=\"checkbox-element\" name=\"radio01\" /><label for=\"checkbox-element\">Interactive Radio 1</label></div>",
-					"react": "<DBTag><DBRadio name=\"radio01\">Interactive Radio 1</DBRadio></DBTag>",
-					"vue": "<DBTag><DBRadio name=\"radio01\">Interactive Radio 1</DBRadio></DBTag>"
-				}
+				"children": [
+					{
+						"name": "link",
+						"props": {
+							"name": "radio01"
+						}
+					}
+				]
 			},
 			{
 				"name": "Interactive Radio 2",
 				"props": {
 					"component": "radio",
-					"identifier": "radio01",
-					"code": {
-						"angular": "<db-tag><db-radio name=\"radio01\">Interactive Radio 2</db-radio></db-tag>",
-						"html": "<div class=\"db-tag\"><input type=\"radio\" class=\"db-checkbox\" id=\"checkbox-element\" name=\"radio01\" /><label for=\"checkbox-element\">Interactive Radio 2</label></div>",
-						"react": "<DBTag><DBRadio name=\"radio01\">Interactive Radio 2</DBRadio></DBTag>",
-						"vue": "<DBTag><DBRadio name=\"radio01\">Interactive Radio 2</DBRadio></DBTag>"
+					"identifier": "radio01"
+				},
+				"children": [
+					{
+						"name": "link",
```

```suggestion
						"name": "radio",
```

I might misunderstand this configuration, but wouldn't this need to be `radio` instead of `link`?

---

Yes, I think it was a copy paste mistake